### ISSUE TITLE
GTK theme not matching as the application's color theme on any chromium-based

### ISSUE THREAD

**Issue by UnifeGi** [bimlas/xfce4-night-mode#13]
This issue, annoys me...
Because, when time reaches to night, the theme doesn't even change. Websites are still light-themed, the interface is also light-themed. And, if I want to change it, I have to go into XFCE's settings go to Appearance and just change the theme, and it works. This time, the websites are dark-themed, and aswell as the interface.
But, when I leave the Appearance window open, and use the icon's plugin (that's this repo) it also changes the theme properly, that also works. But having the Appearance window closed, doesn't change it. So, there's some kind of issue in there. 
And, it may relate to #12.  But even having that in the script, still doesn't work for some reason. Maybe I did something wrong with the script. Just when having the Appearance window opened, worked.

And the plugin is amazing for me because, I don't even have to stare at a light theme in the dark. Or, even in the day, when having the dark theme on. Amazing.. 👍

edit: Seems like it also happens on Firefox... Basically means that, it's not just chromium based.

edit 2: I had to do "gsettings set org.gnome.desktop.interface color-scheme 'prefer-dark" in the terminal, and it works.. so... what I had to do is edit "UserScript" (both of them) in Settings Editor and add in, 
"xfconf-query --channel xfce4-desktop <some text here, this part doesn't matter, because it's an example.>; gsettings set org.gnome.desktop.interface color-scheme 'prefer-light'" 
basically add "; gsettings set org.gnome.desktop.interface color-scheme 'prefer-light'" at the end of line, (if you want multiple commands, add ;). And it's solved, no more light-themed web browser on the dark. Sure, it looks weird to add two lines in every Userscript, and a work-around, but, all that matters, that it's done.

**Comment by UnifeGi** [comment]
I'm just gonna close the issue as it's clogging up in the issues tab...

also, I had it working with themes of applications after I solved it, which is nice.

I was also using Loc-OS, which apparently is a lightweight version of AntiX on Debian 12, (sorry for not mentioning the OS...) which might have had some xfce stuff missing, to make the plugin not work completely.... perhaps dependences..
that's probably why chromium wasn't correcting the system theme before?

other debian-based xfce distros, work without a problem. so, something is going on with Loc-OS...

either way, the issue doesn't have to be opened still.. (but, I'm not too sure...)
